### DISCUSSION TITLE
Keep GPU disabled on USB C charger | Supported Models

### DISCUSSION THREAD

**Discussion by whyredfire** [seerge/g-helper#796] [General]
![image](https://github.com/seerge/g-helper/assets/72072109/1fa1ff23-f539-4676-9085-aaabf3e81dc1)

Can we hide this "Keep GPU disabled on USB-C charger in Optimized mode" feature based on the fact that the laptop supports Optimized mode (MUX switch) or not?

**Comment by seerge** [comment]
@whyredfire it's also stop all apps using GPU checkbox then :) i can hide both, but it's purely esthetical :)

**Reply by whyredfire** [reply]
Ah right, I didn't see the other one. Your call.
Thanks for replying super fast, loved your work!
